### PR TITLE
Update Rosetta models to v1.4.9

### PR DESCRIPTION
**Pull request by psteckler**
Build Rosetta docker for testing new models.

For #9294.

### CODE REVIEW

File: src/app/rosetta/config.json [hunk 1]
```diff
@@ -8,5 +8,262 @@
...
+      {
+        "balance": "0.000000000",
+        "delegate": null,
+        "pk": "B62qrzGZGFwth7HyN7Xb3yNsqzzTDyQp6zruVgSirE2JUqW2sqiYYLr",
+        "sk": null
+      }
+    ],
+    "name": "release",
+    "num_accounts": 250
+  }
```
Comment (bkase) [inline src/app/rosetta/config.json:268]: we'll need to make a new config.json for mainnet too -- I think we should probably build some script that can digest the genesis ledger file, that way we can easily migrate it during a hardfork (cc @deepthiskumar to keep note of when we do a hardfork)

Comment (bkase) [reply src/app/rosetta/config.json:268]: doesn't have to be this PR that we make that config, just flagging as a TODO

File: src/app/rosetta/lib/construction.ml [hunk 2]
```diff
@@ -908,27 +655,5 @@ let router ~get_graphql_uri_or_error ~logger (route : string list) body =
...
-        Errors.Lift.parse ~context:"Request"
-        @@ Construction_submit_request.of_yojson body
-        |> Errors.Lift.wrap
-      in
-      let%bind graphql_uri = get_graphql_uri_or_error () in
-      let%map res =
-        Submit.Real.handle ~env:(Submit.Env.real ~graphql_uri) req
-        |> Errors.Lift.wrap
-      in
-      Construction_submit_response.to_yojson res
```
Comment (bkase) [inline src/app/rosetta/lib/construction.ml:932]: why did you remove these endpoints? I think they're still required: https://www.rosetta-api.org/docs/1.4.9/ConstructionApi.html

Comment (psteckler) [reply src/app/rosetta/lib/construction.ml:932]: yes, you're right, it's just the response types that are changed

I'll restore those

**Comment by psteckler** [comment]
Closing becaue #9341 updates the models.

We'll need to update the Rosetta Docker at some point, which can be done in a new PR.
